datalist-polyfill.js

```diff
@@ -364,7 +375,7 @@
 						} else {
 							$inputList.value = selectValue;
 						}
-						
+						$inputList.dispatchEvent(new CustomEvent("input", {"bubbles":true, "cancelable": true}));
```

Comment:
Two things:
* You can probably use `new Event(...)` instead of `new CustomEvent(...)` - when I see the behaviour of the `"input"` event on-datalist-select in Chrome, it's an `Event`, not a `CustomEvent`
* When Chrome triggers the event, it isn't `cancelable`, so it shouldn't be `cancelable` when polyfilled:
![2018-03-16-002354_667x263_scrot](https://user-images.githubusercontent.com/7784737/37508450-5d47168c-28b0-11e8-9df4-7d70b2e524fe.png)

So, you can probably simplify this down to:
```
$inputList.dispatchEvent(new Event('input', {bubbles: true}));
```